src/ansys/fluent/core/field_data_interfaces.py

```diff
@@ -406,6 +407,99 @@ def is_valid(self, name, respect_data_valid=True):
         return name in self(respect_data_valid)
 
 
+class SurfaceData:
+    """
+    A wrapper class for surface data that enables object-style access
+    to nested dictionary structures.
+
+    This class is used to convert nested dictionary-based surface data
+    into objects for easier attribute-based access.
+
+    Examples
+    --------
+    >>> surface_data = {
+    ...     "surface1": {SurfaceDataType.Vertices: np.array(), FacesConnectivity: np.array()},
```

Comment:
Example with empty array will be misleading. Also user won't directly construct these classes. We can show some real examples in the actual method docstring of FieldData class.